composer.json

```diff
@@ -99,6 +99,8 @@
         "symfony/yaml": "self.version"
     },
     "require-dev": {
+        "amphp/http-client": "^4.0",
+        "amphp/sync": "1.3.0",
```

Comment:
needed until https://github.com/amphp/sync/issues/5 is resolved

Reply:
fixed now.